src/core/InterleavedBufferAttribute.js

```diff
@@ -17,7 +17,7 @@ THREE.InterleavedBufferAttribute.prototype = {
 
 	get length() {
```

Comment:
Right. So we could call it `numVertices`. That would be absolutely clear.

But for some reason, I think I like `count` better, but it is not clear what it is a count of...